### PR TITLE
chore: Add version command

### LINKED ISSUE #66: Add version command to cli help with debugging current version.
Bonus points for if command also prompts you to upgrade if newer version is available.

```
➜  ~ momento --version
error: Found argument '--version' which wasn't expected, or isn't valid in this context

	Did you mean '--verbose'?

	If you tried to supply `--version` as a value rather than a flag, use `-- --version`

USAGE:
    momento --verbose

For more information try --help
➜  ~ momento version
error: Found argument 'version' which wasn't expected, or isn't valid in this context

USAGE:
    momento [OPTIONS] <SUBCOMMAND>

For more information try --help
```

### PR DESCRIPTION
**Pull request by poppoerika**
Also let the user know if their installed momento-cli version is older than the latest version.

```
# momento-cli version is latest
./target/debug/momento version
[2022-03-15T01:25:21Z INFO  momento] Momento v0.1.0

# user has an older version of momento-cli
./target/debug/momento version
[2022-03-15T01:37:02Z INFO  momento] Momento v0.1.2
You have older version of Momento CLI.
Run 'brew upgrade momento-cli' to upgrade to the latest v.0.1.2
```

Closes #66

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -52,6 +52,9 @@ jobs:
         run: |
           chmod +x set_cargo_version.sh
           ./set_cargo_version.sh ${{ steps.semrel.outputs.version }}
+          git add .
```
Comment (bruuuuuuuce) [inline .github/workflows/cd.yml:55]: I think here we might need to set the global git profile like this 

https://github.com/momentohq/momento-cli/blob/main/.github/workflows/cd.yml#L94

File: src/main.rs [hunk 2]
```diff
@@ -12,6 +12,7 @@ pub mod error;
 mod utils;
 
 #[derive(Debug, StructOpt)]
+#[clap(version)]
```
Comment (bruuuuuuuce) [inline src/main.rs:15]: Can we set the name to `momento`. I think the default name is `momento-cli`

**Comment by poppoerika** [comment]
```
./target/debug/momento --version                                       
momento 0.1.0
```

File: .github/workflows/cd.yml [hunk 3]
```diff
@@ -54,6 +54,20 @@ jobs:
...
         shell: bash
 
+      - name: Create version update pull request
+        id: cpr
+        uses: repo-sync/pull-request@v2
+        with:
+          destination_branch: "verions-update/v${{ steps.semrel.outputs.version }}"
+          pr_title: "momento-cli version update ${{ github.ref }}"
+          pr_body: ":crown: *Automated PR*"
+          pr_reviewer: "bruuuuuuuce"
```
Comment (bruuuuuuuce) [inline .github/workflows/cd.yml:64]:
```suggestion
```

File: .github/workflows/cd.yml [hunk 4]
```diff
@@ -54,6 +54,20 @@ jobs:
...
+          pr_assignee: "bruuuuuuuce"
```
Comment (bruuuuuuuce) [inline .github/workflows/cd.yml:65]:
```suggestion
```

File: .github/workflows/cd.yml [hunk 5]
```diff
@@ -54,6 +54,20 @@ jobs:
...
+          pr_label: "automation"
+          pr_draft: false
+          pr_allow_empty: true
+          github_token: ${{ secrets.PRIVATE_REPO_RELEASE_ACCESS }}
```
Comment (bruuuuuuuce) [inline .github/workflows/cd.yml:69]: I don't think we need this token since we are creating branch in same repo

**Review by bruuuuuuuce — APPROVED** [review]
:shipit:

**Pull request merged**